src/components/Ecommerce/Ecommerce.css

```diff
@@ -0,0 +1,44 @@
+body{
```

Comment:
Be careful, this would affect all the solution so you should add a custom container instead of edit the body.